scripts/pack_deb.sh

```diff
@@ -79,6 +79,14 @@ if [[ -n "$martinfolder" ]]; then
     cp -r "$martinfolder/"* "${debworkdir}/opt/ehrenamtskarte/martin"
 fi
 
+# create postinst for db migration
+postinstfile="${debworkdir}/DEBIAN/postinst"
+echo "Creating postinst file in $postinstfile ..."
+touch "$postinstfile"
+chmod 0755 "$postinstfile"
+echo "sudo useradd -r backend" >> "$postinstfile"
```

Comment:
AFAICT you don't need to execute `sudo`. The script should already be running as `root` user.